include/RAJA/policy/openmp/policy.hpp

```diff
@@ -135,12 +139,12 @@ struct omp_collapse_nowait_exec
 };
 
 #if defined(RAJA_ENABLE_TARGET_OPENMP)
-template <size_t Teams>
+template <size_t Threads>
```

Comment:
It's not critical, but I'd prefer this to be something like "ThreadsPerTeam" or something so it's clear it isn't total number of threads.